features/git-town-append/on-parent-branch.feature

```diff
@@ -0,0 +1,61 @@
+Feature: Appending a branch to a parent branch
```

Comment:
I wanted to document the resulting branch hierarchy. For example that appending doesn't insert the new branch between the parent branch and its child, but makes it an independent child. If that's obvious to you, I'm happy to remove this test.